core/templates/dev/head/css/oppia.css

```diff
@@ -4178,3 +4531,18 @@ div.oppia-issues-learner-action {
   max-width: none;
   white-space: nowrap;
 }
+.navbar-fixed-top,.navbar-fixed-bottom {
```

Comment:
As mentioned above: alphabetize, and put all navbar stuff in one place.

Reply:
Done PTAL Thanks!